internal/services/network/network_packet_capture_resource.go

```diff
@@ -374,3 +429,51 @@ func flattenNetworkPacketCaptureFilters(input *[]network.PacketCaptureFilter) []
 
 	return filters
 }
+
+func expandNetworkPacketCaptureScope(input []interface{}) *network.PacketCaptureMachineScope {
+	if len(input) == 0 || input[0] == nil {
+		return nil
+	}
+
+	raw := input[0].(map[string]interface{})
+	output := &network.PacketCaptureMachineScope{}
+
+	if exclude := raw["exclude"].([]interface{}); len(exclude) > 0 {
+		output.Exclude = utils.ExpandStringSlice(exclude)
+	}
+
+	if include := raw["include"].([]interface{}); len(include) > 0 {
+		output.Include = utils.ExpandStringSlice(include)
+	}
+
+	return output
+}
+
+func flattenNetworkPacketCaptureScope(input *network.PacketCaptureMachineScope) []interface{} {
+	if input == nil || (input.Exclude == nil && input.Include == nil) || (len(*input.Exclude) == 0 && len(*input.Include) == 0) {
+		return []interface{}{}
+	}
+
+	return []interface{}{
+		map[string]interface{}{
+			"exclude": flattenNetworkPacketCaptureScopeInstanceIds(input.Exclude),
+			"include": flattenNetworkPacketCaptureScopeInstanceIds(input.Include),
+		},
+	}
+}
+
+func flattenNetworkPacketCaptureScopeInstanceIds(input *[]string) []string {
+	instances := make([]string, 0)
+	if input == nil {
+		return instances
+	}
+
+	for _, instance := range *input {
+		instanceSegments := strings.Split(instance, "/")
+		instanceLastSegment := instanceSegments[len(instanceSegments)-1]
+
+		instances = append(instances, instanceLastSegment)
+	}
+
+	return instances
+}
```

Comment:
What exactly is this doing/what instance Ids are we expecting here, could we potentially use the id parsers here?

Reply:
When we set "0" for new property "include/exclude" (here "0" means VMSS instance ID) in request body, API would return "/subscriptions/xx-xx-xx-xx/resourceGroups/acctestRG-watcher-test016/providers/Microsoft.Compute/virtualMachineScaleSets/acctvmss-test016/virtualMachines/0". So we have to process it while flattening to avoid diff.